### DISCUSSION TITLE
failed to install nocodb-seed [Oct 24]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#4245] [Discord Threads]
> 🙋‍♂️ **johnkaapan**  
##   
![image0](https://cdn.discordapp.com/attachments/661905455894888493/1031287013263036568/unknown.png)

**Comment by o1lab** [comment]
> 🙋‍♂️ **wkw**  
#### this should be resolved in the next version  
#  
> 🙋‍♂️ **wkw**  
#### alternatively, you can clone nocodb/nocodb and run `npm i /path/nocodb/packages/nocodb`  
#  
> 👨‍💻 **pranavxc**  
#### <@897052889468526602> Can you share the error log when you tried `npm i` , it's missing in the screenshot  
#  
> 🙋‍♂️ **wkw**  
#### <@682854952657616902> this s due to some peer dependencies, which i've fixed in develop yst  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1031287013657280622)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
